snmp/datadog_checks/snmp/data/default_profiles/riverbed-interceptor.yaml

```
@@ -0,0 +1,68 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "riverbed"
+
+sysobjectid:
+  - 1.3.6.1.4.1.17163.1.3  # Steelhead Interceptor 9200
+metrics:
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      name: memory.total
+      OID: 1.3.6.1.2.1.25.2.3.1.5.1  # hrStorageSize.1 (hrStorageSize at index 1)
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      name: memory.used
+      OID: 1.3.6.1.2.1.25.2.3.1.6.1  # hrStorageUsed.1 (hrStorageUsed at index 1)
+  - MIB: INTERCEPTOR-MIB
+    symbol:
+      name: cpu.usage
+      OID: 1.3.6.1.4.1.17163.1.3.5.1.4.0
+  - MIB: INTERCEPTOR-MIB
+    symbol:
+      name: riverbed.interceptor.health
+      OID: 1.3.6.1.4.1.17163.1.3.2.2.0
+  - MIB: INTERCEPTOR-MIB
+    symbol:
+      name: riverbed.interceptor.serviceStatus
+      OID: 1.3.6.1.4.1.17163.1.3.2.3.0
```

we don't support scalar string metrics yet
```suggestion
  # - MIB: INTERCEPTOR-MIB
  #  symbol:
  #    name: riverbed.interceptor.serviceStatus
  #    OID: 1.3.6.1.4.1.17163.1.3.2.3.0
  # scalar string metric is not supported yet (keep this metric and this comment in profile until it's fixed)
```